mxcubeweb/core/adapter/sample_view_adapter.py

```diff
@@ -209,6 +209,12 @@ def sample_image_meta_data(self) -> dict:
 
         pixels_per_mm = HWR.beamline.diffractometer.get_pixels_per_mm()
 
+        # We need to get the beam size when changing the image
```

Comment:
So this dependency is as you understand not necessary and we should remove it from the front end